tests/test_decompositions.py

```diff
@@ -129,6 +129,23 @@ def test_clements_random_unitary(self):
             error[k]=np.linalg.norm(V-qrec)
         self.assertAlmostEqual(error.mean() , 0)
 
+
+    def test_clements_eq5_random_unitary(self):
+        self.logTestName()
+        error=np.empty(nsamples)
+        for k in range(nsamples):
+            n=20
+            U=haar_measure(n)
+
+            new_tlist, new_diags = dec.clements_eq5(U)
+
+            U_rec=np.identity(n)
+            for i in new_tlist:
+                U_rec=dec.T(*i) @ U_rec
+            U_rec = np.diag(new_diags) @ U_rec
+            self.assertAlmostEqual(np.linalg.norm(U_rec-U), 0)
```

Comment:
```suggestion
            self.assertAlmostEqual(np.linalg.norm(U_rec-U), 0, delta=self.tol)
```
This ensures that the unittest method uses the SF test suite default tolerance.